OutOfSchool/OutOfSchool.Tests/OrganizationControllerTest.cs

```diff
@@ -1,33 +1,280 @@
+            organizationService = new Mock<IOrganizationService>();
+            organizationController = new OrganizationController(_logger, organizationService.Object);
+            user = new ClaimsPrincipal(new ClaimsIdentity());
+            organizationController.ControllerContext.HttpContext = new DefaultHttpContext { User = user };
+        }
+     
+        [Test]
+        public async Task Get_Organizations_ReturnsOkObjectResult()
+        {
+            // Arrange
+            IEnumerable<OrganizationDTO> organizations = new List<OrganizationDTO>
+            {
+                new OrganizationDTO
+                {
+                      Id = 0b101,
+                      Title = "Title",
+                      Website = "Website",
+                      Facebook = "�������",
+                      Instagram = "���������",
+                      Description = "ererer",
+                      MFO = "123456",
+                      EDRPOU = "12345678",
+                      INPP = "1234567891",
+                      Type = 0,
+                      UserId = "123",
+                },
+            }.ToList();
+
+            organizationService.Setup( it => it.GetAll()).Returns(Task.FromResult(organizations));
+
+            // Act
+            var response = await organizationController.GetOrganizations();
+
+            // Assert
+            Assert.IsInstanceOf<OkObjectResult>(response);
+        }
+
+        [Test]
+        public async Task Get_Organizations_ReturnsBadRequestObjectResult()
+        {
+            // Arrange
+            organizationService.Setup(it => it.GetAll()).Throws(new Exception());
+
+            // Act
+            var response = await organizationController.GetOrganizations();
+
+            // Assert
+            Assert.IsInstanceOf<BadRequestObjectResult>(response);
+        }
+
+        [Test]
+        public async Task Get_OrganizationById_ReturnsOkObjectResult()
+        {
+            // Arrange
+            var organization = new OrganizationDTO
+            {
+                Id = 0b101,
+                Title = "Title",
+                Website = "Website",
+                Facebook = "�������",
+                Instagram = "���������",
+                Description = "ererer",
+                MFO = "123456",
+                EDRPOU = "12345678",
+                INPP = "1234567891",
+                Type = 0,
+                UserId = "123"
+            };       
+          
+            organizationService.Setup(it => it.GetById(organization.Id)).Returns(Task.FromResult(organization));
+
+            // Act
+            var response = await organizationController.GetOrganizationById(organization.Id);
+
+            // Assert
+            Assert.IsInstanceOf<OkObjectResult>(response);
         }
-        [SetUp]
-        public void Setup()
+
+        [Test]
+        public async Task Get_OrganizationById_ReturnsBadRequestObjectResult()
```

Comment:
it would be nice to understand why this test returns bad request
for example from the test name like GetOrganizationById_NonExistentId_ReturnsBadRequestObjectResult